sql/src/test/java/io/crate/operation/reference/sys/node/NodeStatsContextFieldResolverTest.java

```diff
@@ -102,6 +102,17 @@ public void testColumnIdentsResolution() {
     }
 
     @Test
+    public void testNodeStatsContextDoesNotReinitializeItsFieldsForSubColumns() {
+        NodeStatsContext context = resolver.forColumns(ImmutableSet.of(
+            new ColumnIdent(SysNodesTableInfo.Columns.OS.name()),
```

Comment:
SysNodesTableInfo.Columns are already ColumIdents - it's not necessary to re-create new ones